templates/gcds-layout/gcds-layout-core-en.html

```diff
@@ -0,0 +1,60 @@
+---
+{
+	"title": "GCDS Layout - Core",
+	"language": "en",
+	"category": "templates",
+	"altLangPrefix": "gcds-layout-core",
+	"altLangPage": "gcds-layout-core-fr.html",
+	"secondlevel": "false",
+	"dateModified": "2025-10-20",
+	"layout": "gcds-layout",
+	"share": true,
+	"noBreadcrumbs": true,
+	"siteMenuPath": false,
+	"nositesearch": true,
+	"noFooterMain": true,
+	"noFooterContextual": true
```

Comment:
I don't believe this property is necessary, but you can keep it nonetheless. Up to you.